State.go

```diff
@@ -220,6 +220,20 @@ func IsMouseDragging(button int, threshold float64) bool {
 	return C.iggIsMouseDragging(C.int(button), C.float(threshold)) != 0
 }
 
+// GetMouseDragDelta returns the delta from the initial clicking position while the mouse button is pressed or was just released. This is locked and return 0.0f until the mouse moves past a distance threshold at least once (if lockThreshold < -1.0f, uses io.MouseDraggingThreshold).
+func GetMouseDragDelta(button int, lockThreshold float32) Vec2 {
```

Comment:
Getter should be without the `Get` prefix in Go.
See https://golang.org/doc/effective_go#Getters .

Reply:
I missed this, thanks.